tools/ppchcalls_example.txt

```diff
@@ -0,0 +1,157 @@
+Demonstrations of ppchcalls, the Linux/eBPF version.
+
+
+ppchcalls summarizes hcall counts across the system or a specific process,
+with optional latency information. It is very useful for general workload
+characterization, for example:
+
+# ./ppchcalls.py
+Tracing ppc hcalls, printing top 10... Ctrl+C to quit.
+^C[04:59:47]
+PPC HCALL                                        COUNT
+H_IPI                                               26
```

Comment:
It seems that this is very similar to kvmexit tool, do them serve to the same purpose ?

Reply:
Do you have some references for ppc hcalls ?

Reply:
> 

It's different. More details can be found here: https://docs.kernel.org/powerpc/papr_hcalls.html